### PR TITLE
content: optimize tailwindcss in angular

### PR DESCRIPTION
**Pull request by marcjulian**

### CODE REVIEW

File: content/blog/optimize-tailwindcss-in-angular.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+  - Scully
+keywords:
+  - CSS
+  - Utility-First CSS
+authors:
+  - Marc Stammerjohann
+github: https://github.com/notiz-dev/angular-scully-tailwindcss
+---
+
+[Tailwind CSS](https://tailwindcss.com) generates a **large amount** of utility classes for your development build by design. For your Angular application you want the best performance by **only** including the classes you are actually using in your production build. Tailwind got you covered! [PurgeCSS](https://purgecss.com/) is build-in making it easy to remove unused Tailwind styles for your application.
```
Comment (garygrossgarten) [inline content/blog/optimize-tailwindcss-in-angular.md:19]: By design, [Tailwind CSS](https://tailwindcss.com) generates a **large amount** of utility classes for your development build.

File: content/blog/optimize-tailwindcss-in-angular.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+
+You are learning how to [optimize](https://tailwindcss.com/docs/optimizing-for-production) Tailwind CSS in your [Angular](/blog/angular-10-with-tailwindcss) and [Scully](/blog/jamstack-angular-scully-tailwind-css) application.
```
Comment (garygrossgarten) [inline content/blog/optimize-tailwindcss-in-angular.md:21]: You are going to learn...?

File: content/blog/optimize-tailwindcss-in-angular.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+The optimizations in this post have been tested with Angular 11 and Tailwind CSS 2.0, also applies for Angular 10 and Tailwind CSS 1.9.x.
```
Comment (garygrossgarten) [inline content/blog/optimize-tailwindcss-in-angular.md:23]: ???

File: content/blog/optimize-tailwindcss-in-angular.md [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+
+```json
+{
+  "scripts": {
+    "build:prod": "NODE_ENV=production ng build --prod"
+  }
+}
+```
+
+Now run `npm run build:prod` to **only** include used Tailwind CSS utilities in your Angular production build. This works even great in your Scully application.
```
Comment (garygrossgarten) [inline content/blog/optimize-tailwindcss-in-angular.md:88]: This even works great in your Scully application

File: content/blog/optimize-tailwindcss-in-angular.md [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+    extend: {},
+  },
+  variants: {
+    extend: {},
+  },
+  plugins: [],
+};
+```
+
+To take advantage let's migrate our example from above with `purge` as an array to an object using `content` for your template paths.
```
Comment (garygrossgarten) [inline content/blog/optimize-tailwindcss-in-angular.md:115]: to take advantage of ...

File: content/blog/optimize-tailwindcss-in-angular.md [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
+    extend: {},
+  },
+  variants: {
+    extend: {},
+  },
+  plugins: [],
+};
+```
+
+Now you can purge only specific [layers](https://tailwindcss.com/docs/functions-and-directives#layer), `enabled` it manually, go mad with `mode: 'all'` or pass `options` directly to [PurgeCSS](https://purgecss.com/configuration.html#options)
```
Comment (garygrossgarten) [inline content/blog/optimize-tailwindcss-in-angular.md:133]: ???

Comment (garygrossgarten) [reply content/blog/optimize-tailwindcss-in-angular.md:133]: enable it manually or go mad?

File: content/blog/optimize-tailwindcss-in-angular.md [hunk 7]
```diff
@@ -0,0 +1,265 @@
...
+    extend: {},
+  },
+  variants: {
+    extend: {},
+  },
+  plugins: [],
+};
+```
+
+Perfect, now your Angular or Scully applications are optimized for production and you are ready to it to [Firebase Hosting](/blog/firebase-hosting-preview-deploy) or other services.
```
Comment (garygrossgarten) [inline content/blog/optimize-tailwindcss-in-angular.md:265]: ready to deploy it ?

**Review by garygrossgarten — CHANGES_REQUESTED** [review]
see inline comments

**Comment by marcjulian** [comment]
Update the content :)

**Review by garygrossgarten — APPROVED** [review]
👏

**Pull request merged**